src/media/css/elements/nav/index.styl

```diff
@@ -0,0 +1,212 @@
+@require('../../constants/typography');
+@require('constants');
+
+mkt-nav {
+  border-right: 2px solid rgba(0, 0, 0, 0.2);
+  fixed: top 0 left 0;
+  font-size: 18px;
+  height: 100%;
+  overflow-y: scroll;
+  transition: transform cubic-bezier(0.86, 0, 0.07, 1) 333ms;
+  width: 260px;
+  z-index: $zindex--nav-hamburger-root;
+
+  li {
+    margin: 25px 0;
+  }
+
+  a {
+    border-radius: 5px;
+    color: $color--greyscale--white;
+    font-weight: 300;
+    line-height: 1;
+    padding: 9px 9px 6px;
+  }
+  
+  .back {
+    background: #289FFC;
+    border: 2px solid #1C77BD;
+    border-width: 0 2px 0 0;
+    cursor pointer
+    display: block;
+    fixed: top 0 left 0;
+    height: $nav--header-height;
+    overflow: hidden;
+    padding-top: $nav--header-height;
+    width: $nav--width;
+    z-index: $zindex--nav-hamburger-child;
+    &:before {
+      absolute: top (($nav--header-height - 25px) / 2) right 15px;
+      content: url(../img/icon/back.svg);
+      height: 25px;
+      width: 25px
+    }
+  }
+
+}
+
+mkt-nav-root,
+mkt-nav-child {
+  display: block;
+  fixed: top 0 left 0;
+  height: 100%;
+  list-style: none inside;
+  overflow-y: scroll;
+  width: 260px;
+}
+
+mkt-nav-root {
+  background: url(../img/logo/full-white.svg) no-repeat 30px 30px / 182px 35px,
+              linear-gradient(top, #9E1455, #601E78, #052659);
+  height: 100%;
+  padding: 75px 30px 30px 21px;
+  transition: transform cubic-bezier(0.86, 0, 0.07, 1) 333ms;
+  z-index: $zindex--nav-hamburger-root;
+
+  &:after {
+    absolute: top 0 right 0;
+    background: rgba(0, 0, 0, 0.25);
+    content: '';
+    height: 100%;
+    width: 2px;
+    z-index: $zindex--nav-hamburger-root;
+  }
+
+  .active {
+    background: #1994FC;
+    color: #FFF;
+  }
+}
+
+mkt-nav-child {
+  background: #1A95FC;
+  border-right: 2px solid #1070BD;
+  display: none;
+  padding: $nav--header-height 30px 30px;
+  z-index: $zindex--nav-hamburger-child;
+
+  .active {
+    background: #FFF;
+    color: #1994FC;
+  }
+
+  &.visible {
+    display: block;
+  }
+}
+
+{$nav--to-shift} {
+  transform: translateX(0);
+  +nav--visible() {
+    transform: translateX($nav--width);
+  }
+}
+
+{$nav--subnav-to-shift} {
+  transform: translateX(0);
+  +nav--subnav-visible() {
+    transform: translateX($nav--width);
+  }
+}
+
++nav--full-size() {
+  mkt-nav {
+    font-size: 16px;
+    position: static;
+
+    li {
+      margin: 0;
+    }
+
+    .home,
+    .back {
+      display: none;
+    }
+  }
+
+  mkt-nav-root
```

Comment:
Stylistically, probably, but Stylus doesn't care. We use a bit of a hybrid of Stylus syntax and CSS. I wish we didn't.